charts/pgadmin4/values.yaml

```diff
@@ -75,43 +75,17 @@ networkPolicy:
   enabled: true
 
 ingress:
-  ## If true, pgAdmin4 Ingress will be created
```

Comment:
Why to remove comments under `ingress` section? They're actually useful.

Reply:
yeah they are useful, but kinda also to much information. I would rather get more examples going then having all these comments in the value file. The README.md is for general information / guidance. The examples are for reference and the values should just be as common as possible to not give a person the feeling they have to read a book if they already know how helm functions.

Thats my 2 cents on the subject :')